bike.rb

```diff
@@ -0,0 +1,11 @@
+class Bike < Transport
+
+  attr_accessor :max_distance
+
+  def initialize
+    @max_weight = 10
+    @speed = 10
+    @max_distance = 30
+  end
+  
+end
```

Comment:
The red arrow notify u about new row missing in the end of file

Reply:
Added new row in the end of file